shesha/supervisor/saxoPlusManager.py

```diff
@@ -150,30 +150,32 @@ def next(self, *, do_control: bool = True) -> None:
 
             if not (self.iterations % self.frequency_ratio):
                 # Time for first stage start of new WFS exposure. 
-                self.first_stage.reset_wfs_exposure()
```

Comment:
Are you sure it has to be removed ?

Reply:
Same as for saxoPlusScript.example. It can be removed since there is now a keyword in next() method of stageSupervisor to decide whether to stack or not stack the WFS image.